nominees/immunization-calculation-engine.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Immunization Calculation Engine",
   "aliases": [
-    "ICE"
```

Comment:
I'm curious, why are you dropping this alias?

Reply:
Resolved in commit 906b141.